srcpkgs/interception-tools/files/udevmon/run

```diff
@@ -0,0 +1,11 @@
+#!/bin/sh
+OPTS=""
+if [ -r ./conf ]; then
+	. ./conf
+	if [ ! -z "${CONF_FILE}" ]; then
+		OPTS="-c ${CONF_FILE}"
```

Comment:
You should add to OPTS so user can add their own stuff.

Reply:
I'll change this in the next commit.